core/ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/event/EditorSettingsChangedEvent.java

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.ide.api.event;
+
+import com.google.gwt.event.shared.GwtEvent;
+
+/**
+ * Event fired when editor's settings has been changed.
+ *
+ * @author Roman Nikitenko
+ */
+public class EditorSettingsChangedEvent extends GwtEvent<EditorSettingsChangedHandler> {
```

Comment:
FYI, you can combine event and handler into one class, as it did here https://github.com/eclipse/che/blob/projectManager/core/ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/resources/ResourceChangedEvent.java

It is easier for maintaining in the future.